scripts/preparepackages.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const { Listr } = require( 'listr2' );
+const releaseTools = require( '@ckeditor/ckeditor5-dev-release-tools' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+const latestVersion = releaseTools.getLastFromChangelog();
+const versionChangelog = releaseTools.getChangesForVersion( latestVersion );
+
+const tasks = new Listr( [
+	{
+		title: 'Verifying the repository.',
+		task: async () => {
+			const errors = await releaseTools.validateRepositoryToRelease( {
+				version: latestVersion,
+				changes: versionChangelog,
+				branch: 'master'
+			} );
+
+			if ( !errors.length ) {
+				return;
+			}
+
+			return Promise.reject( 'Aborted due to errors.\n' + errors.map( message => `* ${ message }` ).join( '\n' ) );
+		}
+	},
+	{
+		title: 'Running build command.',
+		task: () => {
+			return tools.shExec( 'yarn run build', { async: true, verbosity: 'silent' } );
+		}
+	},
+	{
+		title: 'Updating the `#version` field.',
+		task: () => {
+			return releaseTools.updateVersions( {
+				version: latestVersion
+			} );
+		}
+	},
+	{
+		title: 'Creating the `ckeditor5-vue2` package in the release directory.',
+		task: () => {
+			return releaseTools.prepareRepository( {
+				outputDirectory: 'release',
+				rootPackageJson: require( '../package.json' )
+			} );
+		}
+	},
+	{
+		title: 'Cleaning-up.',
+		task: () => {
+			return releaseTools.cleanUpPackages( {
+				packagesDirectory: 'release',
+				packageJsonFieldsToRemove: [
+					// TODO: Preserve `scripts.postinstall`.
+					// See: https://github.com/ckeditor/ckeditor5/issues/14318.
+					'devDependencies'
+				]
+			} );
```

Comment:
Do we need the `postinstall` script in a package published on npm?

Reply:
We are including the `postinsdtall` hook in the integration packages, like:
https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code
https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code

So I added it to keep the convention. 

I guess we have to ask @pomek about it when he gets back, because I was not able to figure out if it's required or not.